app/helpers/samples_helper.rb

```diff
@@ -49,6 +49,7 @@ def generate_sample_list_csv(formatted_samples)
   end
 
   # Load bulk metadata from a CSV file
+  # TODO: Succeeded by Metadatum#load_csv_from_s3 for new Metadatum objects.
```

Comment:
nit: What is the TODO? convert and merge for old data?

Reply:
more like a NOTE than a TODO

Reply:
Can you move it to Note:  then ? 
TODOs are typically used for tracking.